test/slider-test.jsx

```diff
@@ -3,10 +3,21 @@ import React from 'react';
 import sinon from 'sinon';
 import { assert } from 'chai';
 import has from 'has';
+import aphroditeInterface from 'react-with-styles-interface-aphrodite';
+import ReactWithStylesThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
 
-import Slider from '../src/Slider';
+import Slider, { Rheostat } from '../src/Slider';
 import { KEYS } from '../lib/constants/SliderConstants';
 
+import DefaultTheme from '../stories/theme/default';
+
+// Registering the default theme more than once clobbers all of the
+// previously created styles, which causes errors to happen. This can happen
+// if this file appears in more than one bundle on the same page. To avoid
+// this, we are using a global cache.
+ReactWithStylesThemedStyleSheet.registerDefaultTheme(DefaultTheme);
+ReactWithStylesThemedStyleSheet.registerInterface(aphroditeInterface);
```

Comment:
This seems to be repeated. Perhaps it should go in a module that both places can import?